tokens/connect/metamask.md

```diff
@@ -38,15 +38,13 @@ Learn [how to integrate a Connect MetaMask button](/builders/interact/metamask-d
 <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/ywpc1UwpIyg' frameborder='0' allowfullscreen></iframe></div>
 <style>.caption { font-family: Open Sans, sans-serif; font-size: 0.9em; color: rgba(170, 170, 170, 1); font-style: italic; letter-spacing: 0px; position: relative;}</style><div class='caption'>In this video, we'll show you how to connect your MetaMask wallet to the Moonriver network</a></div>
 
-
```

Comment:
I'm guessing the Connect to Moonbeam button is added in the other PR, just worth mentioning to make sure :)

Reply:
yes it is!